purchase_report_custom_hls/report/purchase_order_report_templates.xml

```diff
@@ -1,19 +1,67 @@
     >
         <xpath expr="//t[@t-set='information_block']" position="attributes">
             <attribute name="groups">purchase_report_custom_hls.group_show_purchase_shipping_address</attribute>
         </xpath>
-        <xpath expr="//div[@t-if='o.date_order']" position="before">
-            <div t-if="o.payment_term_id">
-                <strong>Payment Terms:</strong>
-                <br />
-                <span t-field="o.payment_term_id"/>
+
+        <xpath expr="//p[@t-field='o.date_order']" position="attributes">
+            <attribute name="t-options">{"widget": "date"}</attribute>
+        </xpath>
+        <xpath expr="//div[@id='informations']" position="after">
+            <div id="second_informations" class="row mt32 mb32">
+                <div t-if="o.etd_date" class="col-3">
+                    <strong>ETD:</strong>
+                    <br />
+                    <span t-field="o.etd_date"/>
+                </div>
+                <div t-if="o.eta_date" class="col-3">
+                    <strong>ETA:</strong>
+                    <br />
+                    <span t-field="o.eta_date"/>
+                </div>
+                <div t-if="o.payment_term_id" class="col-3">
+                    <strong>Payment Terms:</strong>
+                    <br />
+                    <span t-field="o.payment_term_id"/>
+                </div>
             </div>
         </xpath>
+        <xpath expr="//span[@t-field='line.product_uom.name']" position="after">
+            <br/>
+            <span t-field="line.secondary_uom_qty"/>
+            <span t-field="line.secondary_uom_id" groups="uom.group_uom"/>
+        </xpath>
+
+        <xpath expr="//table[@class='table table-sm o_main_table']/thead/tr/th[2]" position="attributes">
+            <attribute name="t-if">o.display_tax</attribute>
+        </xpath>
+        <xpath expr="//table[@class='table table-sm o_main_table']/tbody/tr/td[2]" position="attributes">
+            <attribute name="t-if">o.display_tax</attribute>
+        </xpath>
+        <xpath expr="//div[@class='col-4']/table[@class='table table-sm']/tr[2]" position="attributes">
+            <attribute name="t-if">o.display_tax</attribute>
+        </xpath>
+
+        <xpath expr="//table[@class='table table-sm o_main_table']/thead/tr/th[3]" position="attributes">
+            <attribute name="t-if">o.display_date_req</attribute>
+        </xpath>
+        <xpath expr="//table[@class='table table-sm o_main_table']/tbody/tr/td[3]" position="attributes">
+            <attribute name="t-if">o.display_date_req</attribute>
+        </xpath>
+
+        <xpath expr="//span[@t-field='o.amount_tax']/../.."  position="after">
+            <tr>
+                <td>Quantity Total</td>
+                <td class="text-right">
+                    <span t-field="o.quantity_total"/>
+                    <span t-field="o.order_line[0].product_uom"/>
+                </td>
+            </tr>
+        </xpath>
```

Comment:
I personally think it's an odd place to show the total quantities.  Unless the customer specifically requests that the field show the information here, I'd consider putting it in the header part next to the payment terms, perhaps.  We may go as it is for now, but please check with the customer.